tests/wpt.js

```diff
@@ -0,0 +1,78 @@
+const WebPageTest = require('webpagetest');
+const fs = require('fs');
+const path = require('path');
+const { argv } = require('node:process');
+
+const is_direct_run = require.main === module;
+
+const wptServer = 'www.webpagetest.org';
```

Comment:
@rviscomi @pmeenan
We need to change a server and key (value in Github secrets) to start using httparchive private instance.

Reply:
@pmeenan And I would appreciate a general review of this wpt script to avoid discrepancies with the monthly crawler configuration.